policy/modules/kernel/kernel.if

```diff
@@ -4283,3 +4284,19 @@ interface(`kernel_unlabeled_entry_type',`
 	allow $1 unlabeled_t:file { mmap_exec_file_perms ioctl lock };
 ')
 
+
```

Comment:
This is just a cosmetic detail, but there is an extra empty line being added here. And there seems to be also an extra empty line being added to the end of file.

Reply:
Updated.